osu.Game/Tests/FlakyTestAttribute.cs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+#nullable disable
```

Comment:
Why does this keep happening? Do you have some automatic rider rule turned or or something?

Reply:
I think it's a rider bug when using the "move to new file" suggestion.